libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -238,22 +245,22 @@ - (BOOL)isLaunching
 
 - (NSString *)connectedAppId
 {
-    return [SFUserAccountManager sharedInstance].oauthClientId;
+    return self.appConfig.remoteAccessConsumerKey;
```

Comment:
These deprecate properties now point directly into the app config, for backward compatibility.